src/Razor/src/Microsoft.VisualStudio.LanguageServices.Razor/ProjectSystem/DefaultWindowsRazorProjectHost.cs

```diff
@@ -162,7 +162,12 @@ protected override async Task HandleProjectChangeAsync(string sliceDimensions, I
             return false;
         }
 
-        configuration = new(languageVersion, configurationItem.Key, extensions, ForceRuntimeCodeGeneration: forceRuntimeCodeGeneration);
+        configuration = new(
+            languageVersion,
+            configurationItem.Key,
+            extensions,
+            new LanguageServerFlags(forceRuntimeCodeGeneration));
```

Comment:
Should also pass the correct flag value for configurations used in `FallbackProjectManager` and `FallbackWindowsRazorProjectHost`, as they would affect closed document generation in VS.

Reply:
Good catch